example/thanos-compact.yaml

```diff
@@ -6,4 +6,4 @@ spec:
   resources:
     requests:
       storage: 250Gi
-  storageClassName: ebs-gp2
+  storageClassName: ebs-gp3
```

Comment:
What?

Reply:
it's an example :D so I used to check merged config